src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewsFeatureProvider.cs

```diff
@@ -5,9 +5,9 @@
 using Microsoft.AspNetCore.Mvc.Razor;
 using Microsoft.AspNetCore.Mvc.Razor.Compilation;
 using Microsoft.Extensions.Options;
-using OrchardCore.DisplayManagement.FileProviders;
 using OrchardCore.DisplayManagement.Liquid.Internal;
 using OrchardCore.Environment.Extensions;
+using OrchardCore.Mvc.FileProviders;
```

Comment:
I see many new `usings` without code changes, expected?

Reply:
Yes for this one because i moved the already used `GetViewFilePaths()` utility which i use here.

In other places it's just some using re-ordering after having done and then reverted some changes. Many times i revert some changes saying to me that they are not related and need another PR.